### PR TITLE
provide link to attach propales to interventions and a way to restrict selection of links to a specific thirdparty

### PR DESCRIPTION
**Pull request by bafbes**

### CODE REVIEW

File: htdocs/core/class/html.form.class.php [hunk 1]
```diff
@@ -5024,7 +5024,7 @@ function showLinkedObjectBlock($object)
      *  @param	array			$restrictlinksto	Restrict links to some elements, for exemple array('order') or array('supplier_order')
      *  @return	int									<0 if KO, >0 if OK
      */
-    function showLinkToObjectBlock($object, $restrictlinksto=array())
+    function showLinkToObjectBlock($object, $restrictlinksto=array(),$fk_soc=0)
```
Comment (eldy) [inline htdocs/core/class/html.form.class.php]: Adding a parameter must also be added into the code comments of function.
Also we should not need this parameter, the $object->fk_soc contains already the company

File: htdocs/comm/propal.php [hunk 2]
```diff
@@ -2256,6 +2256,12 @@
 		$linktoelem = $form->showLinkToObjectBlock($object);
 		if ($linktoelem) print '<br>'.$linktoelem;
 
+        // Show links to link elements
```
Comment (eldy) [inline htdocs/comm/propal.php]: We should not need this. All links we want to see should be reported into showLinkToObjectBlock.
So if a special cas must be managed, it should be inside function

**Comment by bafbes** [comment]
conflicts solved

**Comment by eldy** [comment]
Somthing seems wrong with PR submission.
817 files modified compared to develop branch !  So i Close it.

**Comment by bafbes** [comment]
I will correct it and recreate it from scratch later.

**Pull request closed without merging**